packages/artifact/package-lock.json

```diff
@@ -4,6 +4,11 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "@actions/core": {
```

Comment:
@konradpabjan lerna doesn't typically check in sibling projects. This will get removed when other developers run commands, and they will have to manually update or fail the build. for example : https://github.com/actions/toolkit/pull/428/files

Can we run `lerna bootstrap` rather then npm ci here: https://github.com/actions/toolkit/pull/421/files#diff-c0a0a8d36f11bf33d1a7ea8e7cda953dR36

Reply:
Thanks for the heads up @thboop !

I was wondering why sometimes locally `package-lock.json` would change but then when running here it wouldn't. That explains it. I'll work on addressing this in a subsequent PR